CMakeLists.txt

```diff
@@ -331,10 +331,7 @@ install_helper(
   FILES ${CMAKE_SOURCE_DIR}/src/man/nvim.1
   DESTINATION ${CMAKE_INSTALL_MANDIR}/man1)
 
-option(ENABLE_LIBICONV "enable libiconv" ON)
-if(ENABLE_LIBICONV)
-  find_package(Iconv REQUIRED)
-endif()
+find_package(Iconv REQUIRED)
```

Comment:
You can move this to `src/nvim/CMakeLists.txt`, only reason it was here is because `Iconv_FOUND` was needed by `cmake.config/CMakeLists.txt`.

Reply:
Thanks!